tools/ami-cleanup/src/cmd/main.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/alecthomas/kingpin/v2"
+
+	"github.com/gravitational/shared-workflows/tools/ami-cleanup/internal"
+)
+
+var (
+	Version = "0.0.0-dev"
+)
+
+func main() {
+	kingpin.Version(Version)
+	dryRun := kingpin.Flag("dry-run", "set to only report the expected changes without actually performing them").Bool()
```

Comment:
Do we really need kingpin for a program that takes a single flag? Less dependencies means less things to patch and keep up to date. Would the `flag` package suffice here?

Reply:
> Do we really need kingpin for a program that takes a single flag?

No, but the last couple of times I brought kingpin vs flag up in slack [the opposite argument was made](https://gravitational.slack.com/archives/C0351AUPJ8K/p1673392009796309?thread_ts=1673053788.975829&cid=C0351AUPJ8K), [or wasn't addressed](https://gravitational.slack.com/archives/C0351AUPJ8K/p1697840072351889).

Do you want me to change this to flag?

Reply:
No strong preference, totally up to you.

I now realize I am the one who made the opposite argument last time. Oops!

I suppose that reflects my lack of a strong opinion. I'm okay with whatever you decide.

In my defense, the last PR was in gravitational/teleport where we already have a kingpin dependency. This is a separate repo with a much smaller set of dependencies and also happens to run some of our sensitive workflows, so it seems reasonable to keep dependencies to a minimum.